Configuration/TCA/tx_tea_domain_model_product_tea.php

```diff
@@ -19,12 +19,14 @@
         'translationSource' => 'l10n_source',
     ],
     'types' => [
-        '1' => ['showitem' => '
+        '1' => [
+            'showitem' => '
             --div--;LLL:EXT:core/Resources/Private/Language/Form/locallang_tabs.xlf:general,
-                title, description, image,
+                title, description, image, owner,
             --div--;LLL:EXT:tea/Resources/Private/Language/locallang_db.xlf:tx_tea_domain_model_product_tea.tabs.access,
                 --palette--;;access,
-        '],
+        ',
+        ],
```

Comment:
That indentation looks strange to me. The indentation of closing `'` doesn't seem related to the line containing the opening one. It is on the same level as the outer `[]`.

Reply:
I'll undo this formatting change and move it (in a cleaned-up version) to a separate PR. I think this was some fallout from me having autoformatted the file.